src/js/components/TextInput/stories/Custom.js

```diff
@@ -74,31 +74,35 @@ const folks = [
   },
 ];
 
-class CustomSuggestionsTextInput extends Component {
-  state = { value: '', suggestionOpen: false, suggestedFolks: [] };
+const CustomSuggestionsTextInput = () => {
+  const [value, setValue] = useState('initialState');
```

Comment:
```suggestion
  const [value, setValue] = useState();
```
This should leave room for the `placeholder` to kick in